docs/src/xhtml/getstarted/index.xhtml

```diff
@@ -50,6 +51,50 @@
 						</script>
 					</figure>
 					-->
+				<h3>NPM version</h3>
+				<p>
+					Tradeshift UI is also available as an
+					<a href="https://www.npmjs.com/package/@tradeshift/tradeshift-ui" target="_blank">
+						npm package </a
+					>. You can simply install it in your project executing the next command
+				</p>
+				<figure data-ts="DoxMarkup">
+					<script type="text/html">
+						npm i @tradeshift/tradeshift-ui --save
+					</script>
+				</figure>
+				<p>
+					then don't forget to import <code>ts.js</code> and <code>ts.css</code> in your main
+					application entrypoint
+				</p>
+				<figure data-ts="DoxMarkup">
+					<script type="text/html">
+						require('@tradeshift/tradeshift-ui');
+						require('@tradeshift/tradeshift-ui/ts.css');
+					</script>
+				</figure>
+				<p>
+					there is a style helpers in the package, so you can import it somewhere in your
+					<code>shared styles module</code>
+				</p>
+				<figure data-ts="DoxMarkup">
+					<script type="text/html">
+						@import '~@tradeshift/tradeshift-ui/ts-runtime';
+					</script>
+				</figure>
+				<p>
+					For <code>Typescript</code> projects, we also supply the library with type definitions.
+					Take a look at
+					<a href="https://www.npmjs.com/package/@tradeshift/tradeshift-ui-types" target="_blank">
+						this package </a
+					>and add it to your Typescript application
+				</p>
+				<figure data-ts="DoxMarkup">
+					<script type="text/html">
+						npm i @tradeshift/tradeshift-ui-types --save-dev
```

Comment:
I saw a PR recently that needed this in dependencies - https://github.com/Tradeshift/fintech-seller-dashboard/pull/150

Reply:
that's true. I totally forgot about it. thank you!

Reply:
fixed (lazy enough to fix on the screenshot :) )